bin/run-bisect

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env bash
+
+# For this to work properly, your configuration file must set
+#   USE_CLUSTER_SPARK = False
+#   SPARK_COMMIT_ID = os.environ["SPARK_COMMIT_ID"]
+
+FWDIR="$(cd "`dirname "$0"`/../"; pwd)"
+
+if [ "$#" -ne 4 ]; then
+  echo "Test every Nth commit between two commits"
+  echo "Usage: oldest_commit newest_commit N config_file"
+  exit 1
+fi
+
+oldestCommit="$1"
+newestCommit="$2"
+NR="$3"
+configFile="$4"
+
+# Find the SHAs of every NRth commit between two git commits:
+cd "$FWDIR/spark-build-cache/master"
```

Comment:
I think we need a check here in case `./bin/run` hasn't been run before and there is no build cache.